static/app/utils/replays/replayDataUtils.tsx

```diff
@@ -222,39 +220,58 @@ export function spansFactory(spans: ReplaySpan[]) {
  */
 export function replayTimestamps(
   replayRecord: ReplayRecord,
-  rrwebEvents: RecordingEvent[],
-  rawCrumbs: ReplayCrumb[],
-  rawSpanData: ReplaySpan[]
+  rawErrors: any[],
+  {
+    rawRRWebEvents,
+    rawBreadcrumbs,
+    rawNetworkSpans,
+    rawMemorySpans,
+  }: {
+    rawBreadcrumbs: any[];
+    rawMemorySpans: any[];
+    rawNetworkSpans: any[];
+    rawRRWebEvents: any[];
+  }
 ) {
```

Comment:
We didn't include erorrs before, so that's added as an arg.

And instead of one spans array, we can accept both network and memory spans separately, so we can split attachements early and not have to recombine things.